src/test/unit/requests/publicationRequests.test.ts

```diff
@@ -61,53 +59,128 @@ describe('Get publication by court id', () => {
 
   it('should return data on successful get', async () => {
-    expect(await publicationRequests.getPublicationsByCourt(valid, true)).toBe(successResponse.data);
+    expect(await pubRequests.getPublicationsByCourt(valid, true)).toBe(successResponse.data);
   });
 
   it('should handle error response from returned service returning empty array', async () => {
-    expect(await publicationRequests.getPublicationsByCourt(invalid, true)).toStrictEqual([]);
+    expect(await pubRequests.getPublicationsByCourt(invalid, true)).toStrictEqual([]);
   });
 
   it('should handle error request from returned service returning empty array', async () => {
-    expect(await publicationRequests.getPublicationsByCourt('test', true)).toStrictEqual([]);
+    expect(await pubRequests.getPublicationsByCourt('test', true)).toStrictEqual([]);
   });
 
   it('should handle error request from returned service returning empty array', async () => {
-    expect(await publicationRequests.getPublicationsByCourt('error', true)).toStrictEqual([]);
+    expect(await pubRequests.getPublicationsByCourt('error', true)).toStrictEqual([]);
   });
 });
 
 describe('get Publication request', () => {
-  dataManagementStub.withArgs('/publication/search/0').resolves({data: pubs});
+  dataManagementStub.withArgs('/publication/courtId/0').resolves({data: pubs});
   it('should return list of publications if verified', async () => {
-    const pubReq = await pubRequests.getListOfPubs(0, true);
+    const pubReq = await pubRequests.getPublicationsByCourt('0', true);
     expect(pubReq.length).toBe(totalCases);
   });
 
   it('should return list of publications if unverified', async () => {
-    const pubReq = await pubRequests.getListOfPubs(0, false);
+    const pubReq = await pubRequests.getPublicationsByCourt('0', false);
     expect(pubReq.length).toBe(totalCases);
   });
 
   it('should contain a publication', async () => {
-    const pubReq = await pubRequests.getListOfPubs(0, true);
+    const pubReq = await pubRequests.getPublicationsByCourt('0', true);
     expect(pubReq.some(e => e.provenance === 'NOT_A_PDF')).toBeTruthy();
   });
 
   it('should send an error request to the log if error request exists', async ()=> {
     dataManagementStub.withArgs('/publication/search/x').resolves(Promise.reject(errorRequest));
-    expect(await pubRequests.getListOfPubs('x', true)).toBe(null);
+    expect(await pubRequests.getPublicationsByCourt('x', true)).toMatchObject([]);
   });
 
   it('should send an error to the log if error message exists and error request does not exist', async () => {
     dataManagementStub.withArgs('/publication/search/y').resolves(Promise.reject(errorMessage));
-    const message = await pubRequests.getListOfPubs('y', true);
-    expect(message).toBe(null);
+    dataManagementStub.withArgs('/publication/search/x').resolves(Promise.reject(errorRequest));
+    expect(await pubRequests.getPublicationsByCourt('x', true)).toMatchObject([]);
```

Comment:
these need deleting anyway, but if you find them neccessary the courtId endpoint is no longer /search, its the /courtId as you have above so i dont know which youre testing here but needs looking at